src/python/pants/notes/2.13.x.md

```diff
@@ -1,5 +1,101 @@
 # 2.13.x Release Series
 
+## 2.13.0rc0 (Jul 20, 2022)
+
+### New Features
+
+* allow JVM sources to have files dependencies (Cherry-pick of #16203) ([#16236](https://github.com/pantsbuild/pants/pull/16236))
+
+* Add [python].default_run_goal_use_sandbox (Cherry-pick of #16239) ([#16253](https://github.com/pantsbuild/pants/pull/16253))
+
+* Add `include_sources` to `pex_binary` target (Cherry-pick of #16215) ([#16252](https://github.com/pantsbuild/pants/pull/16252))
+
+* Add support for extra env variables in go tests (Cherry-pick of #16013) ([#16067](https://github.com/pantsbuild/pants/pull/16067))
+
+### User API Changes
+
+* Ban `../` in `source` and `sources` fields (Cherry-pick of #16227) ([#16237](https://github.com/pantsbuild/pants/pull/16237))
+
+* Upgrade Pex to 2.1.99. (Cherry-pick of #16184) ([#16225](https://github.com/pantsbuild/pants/pull/16225))
+
+* Correct deprecation warning to 2.15 (Cherry-pick of #16056) ([#16218](https://github.com/pantsbuild/pants/pull/16218))
+
+* Upgrade Pex to 2.1.98. (#16173) ([#16178](https://github.com/pantsbuild/pants/pull/16178))
+
+* Upgrade Pex to 2.1.96 and default version of black to 22.6.0. (Cherry-picks of #16137 & #15960) ([#16143](https://github.com/pantsbuild/pants/pull/16143))
+
+* Revert "Upgrade Pex to 2.1.95 and default version of black to 22.6.0." ([#16128](https://github.com/pantsbuild/pants/pull/16128))
+
+* Upgrade Pex to 2.1.95 and default version of black to 22.6.0. (Cherry-picks of #16110 & #15960) ([#16116](https://github.com/pantsbuild/pants/pull/16116))
```

Comment:
Imo this should be left off - it's noise to users

Reply:
Moved to internal comment.